CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-Extension "Job Offers" for Contao Open Source CMS
+Extension "Offers" for Contao Open Source CMS
 ========
+X.Y.Z - YYYY-MM-DD
```

Comment:
Fix the version & date

Reply:
Fixed in 9dd41f1febc20264f9a9ad0c4e4df8afc3118e04